src/components/Receiving/LineDetails.js

```diff
@@ -23,6 +23,7 @@ const LineDetails = ({
   lineItems,
   locationsOptions,
   onChangeField,
+  onChangeBarcode,
```

Comment:
does it make sense to pass that prop? IMHO, it could live here, within the field. It has access to all required props, as far as I see. It's arguable though, maybe additional input from other guys is required , cc @AndreiShumski2 @NikitaSedyx

Reply:
100%